### ISSUE TITLE
Add api.createDiscussion

### ISSUE THREAD

**Issue by demurgos** [insa-frif/palantiri-interfaces#1]
We need to support the creation of discussions.

``` ts
createDiscussion(contacts: Array<AccountReference | AccountGlobalId>, options: any): Thenable<Discussion>
```

This function would attempt to create a completely new discussion on the service with the supplied contacts and the localAccount.
If the service does not support the direct addition of contacts, it should send an invitation to the contacts.

Options should be better defined. Here are some ideas:
- `name` (`string`): tries to set the name of the discussion during its creation
- `description` (`string`): tries to set the description of the discussion during its creation

**Comment by Sn0wFox** [comment]
In fact, in some protocols the newly created Discussions are not really here, just ghosts that the user can use to eventually send a message. It's when the first message is sent that the discussion is really created and obtains an ID.
Maybe we can do the same trick at this level, and at higher ones too.

**Comment by demurgos** [comment]
This is fixed with `0.3.2`:
https://github.com/insa-frif/palantiri-interfaces/commit/45fc4986e6140c00a72386130eb142c33f5312cd
